openlibrary/components/IdentifiersInput.vue

```diff
@@ -29,15 +29,15 @@
     </tr>
       <template v-for="(value, name) in assignedIdentifiers">
         <tr :key="name" v-if="value && !saveIdentifiersAsList">
-          <td>{{ identifierConfigsByKey[name].label }}</td>
+          <td>{{ identifierConfigsByKey[name] ? identifierConfigsByKey[name].label : name }}</td>
```

Comment:
We can use the `?.` operator here to make this a bit more concise:

```suggestion
          <td>{{ identifierConfigsByKey[name]?.label ?? name }}</td>
```